client/src/reducers/DirectoryReducer.js

```diff
@@ -17,6 +17,11 @@ const DirectoryReducer = (state = initialState, action) => {
       newState = Object.assign({}, state, { loading: false });
       newState[action.role].push(action.user);
       return newState;
+    case 'EDIT_USER_SUCCESS':
+      return {
+        ...state,
+        loading: false,
```

Comment:
The following code will find the correct user in the front end and update it correctly.
newState = Object.assign({}, state, { loading: false });
newState[action.role].find(user => user.id === action.userId)
return newState;

Reply:
A quick question... so changing the value of `loading` doesn't refresh the page, why is that?

Reply:
Also it doesn't refresh for some reason :(